controllers/user.js

```diff
@@ -9,7 +9,9 @@ const UNSUB_REASONS = require("../unsubscribe_reasons");
 const sha1 = require("../sha1-utils");
 const TIPS = require("../tips");
 
+
 async function add(req, res) {
+  const SERVER_URL = EmailUtils.getServerUrl();
```

Comment:
Aren’t we shimming this into `app.res.locals.SERVER_URL`?
Or do we need another method for emails?

Reply:
We evidently need another method for emails / some handlebars magic sauce but even like this, SERVER_URL doesn't seem to make it all the way to report_password_tips.hbs. When I inspect the password tip icons from gmail, 'localhost' is nowhere to be found amongst the gmail gobblygook, but is definitely there for the other images.

Reply:
I added https://github.com/mozilla/blurts-server/pull/377/commits/79023b089224b19bf8e60d897967145b694c4c57 to this PR which moves `SERVER_URL` context assignment into `sendEmail` so it will be available for *ALL* email templates.